spyder/plugins/updatemanager/workers.py

```diff
@@ -9,10 +9,12 @@
 import logging
```

Comment:
Please add before the `datetime` import this line

```python
from __future__ import annonations
```

That's necessary to use the `|` operator below in Python 3.8.